### PR TITLE
fix help text for dev object cmds

### PR DESCRIPTION
**Pull request by sunkickr**
## Description

fix help text for dev object cmds

## 🎟 Issue(s)

Related #XXX

## 🧪 Functional Testing

> List the functional testing steps to confirm this feature or fix.

## 📸 Screenshots

> Add screenshots to illustrate the validity of these changes.

## 📋 Checklist

- [X] Rebased from the main (or release if patching) branch (before testing)
- [X] Ran `make test` before taking out of draft
- [X] Ran `make lint` before taking out of draft
- [ ] Added/updated applicable tests
- [ ] Tested against [Astro-API](https://github.com/astronomer/astro/) (if necessary).
- [ ] Tested against [Houston-API](https://github.com/astronomer/houston-api/) and [Astronomer](https://github.com/astronomer/astronomer/) (if necessary).
- [ ] Communicated to/tagged owners of respective clients potentially impacted by these changes.
- [ ] Updated any related [documentation](https://github.com/astronomer/docs/)

### CODE REVIEW

File: cmd/airflow.go [hunk 1]
```diff
@@ -354,27 +354,27 @@ func newAirflowObjectRootCmd() *cobra.Command {
 func newObjectImportCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "import",
-		Short: "Create and update local Airflow objects from an Airflow Settings file",
-		Long:  "This command will create all connections, variables, and pools in an Airflow Settings file locally. Airflow must be running locally for this command to work",
+		Short: "Create and update local Airflow objects from an Airflow Settings yaml file",
+		Long:  "This command will create all connections, variables, and pools in an Airflow Settings yaml file locally. Airflow must be running locally for this command to work",
```
Comment (aspain) [inline cmd/airflow.go:358]:
```suggestion
		Long:  "This command will create all connections, variables, and pools in a local Airflow instance from a local Airflow Settings yaml file. Airflow must be running locally for this command to work",
```

File: cmd/airflow.go [hunk 2]
```diff
@@ -354,27 +354,27 @@ func newAirflowObjectRootCmd() *cobra.Command {
...
+	cmd.Flags().StringVarP(&settingsFile, "settings-file", "s", "airflow_settings.yaml", "Settings yaml file to export objects to")
 	return cmd
 }
 
 func newObjectExportCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "export",
-		Short: "Export all Airflow objects to an airflow settings or env file. Does not overwrite objects that already exist in the file",
-		Long:  "This command will export all Airflow objects to an airflow settings or env file(--env-export). Objects already in the file will not be over written. Airflow must be running locally for this command to work",
+		Short: "Export all Airflow objects to an Airflow Settings yaml or env file(--env-export). Does not overwrite objects that already exist in the file",
```
Comment (aspain) [inline cmd/airflow.go:376]:
```suggestion
		Short: "Export all Airflow objects from a local Airflow instance to an Airflow Settings yaml or env file(--env-export). Does not overwrite objects that already exist in the file",
```

File: cmd/airflow.go [hunk 3]
```diff
@@ -354,27 +354,27 @@ func newAirflowObjectRootCmd() *cobra.Command {
...
+		Long:  "This command will export all Airflow objects to an Airflow Settings yaml or env file(--env-export). Objects already in the file will not be over written. Airflow must be running locally for this command to work",
```
Comment (aspain) [inline cmd/airflow.go:377]:
```suggestion
		Long:  "This command will export all Airflow objects from a local Airflow instance to an Airflow Settings yaml or env file(--env-export). Objects already in the file will not be overwritten. Airflow must be running locally for this command to work",
```

**Comment by kushalmalani** [comment]
Seems like @jwitz also has a PR to update these typos - https://github.com/astronomer/astro-cli/pull/777

**Comment by sunkickr** [comment]
We will use jakes PR

**Pull request closed without merging**